data/parties/planet-insomnia/pi10.toml

```diff
@@ -0,0 +1,15 @@
+slug = "planet-insomnia-10"
+title = "Planet Insomnia 10"
+series_slug = "planet-insomnia"
+start_on = 2001-04-06
+end_on = 2001-04-08
+seats = 400
```

Comment:
I'm not sure about this anymore, but it should come pretty close.

Reply:
I guess that's good enough until we find better numbers.